pipeline-notebooks/pipeline-section.ipynb

```diff
@@ -1007,7 +1008,10 @@
     "            section_elements = sec_document.get_section_narrative(regex_enum)\n",
     "            results[f\"REGEX_{i}\"] = section_elements\n",
     "    if response_type == \"application/json\":\n",
-    "        return {section:convert_to_isd(section_narrative) for section, section_narrative in results.items()}\n",
+    "        if response_schema == \"labelstudio\":\n",
+    "            return {section:stage_for_label_studio(section_narrative) for section, section_narrative in results.items()}\n",
+    "        else:\n",
+    "            return {section:convert_to_isd(section_narrative) for section, section_narrative in results.items()}\n",
```

Comment:
let's raise a ValueError if `response_schema` not in "isd", "labelstudio". actually while we're at, might as well make those constants.

Reply:
Done.